python/repositories.bzl

```diff
@@ -148,6 +148,16 @@ cc_library(
     ],
 )
 
+cc_import(
+    name = "libpython",
+    hdrs = [":includes"],
+    shared_library = glob([
```

Comment:
Can you make this a select so it can be more explicit? Shouldn't all toolchains be consistent in exposing this?

Reply:
Fixed in 043f2e388dcf6854260253ef19e7079d32babbbe.

Reply:
Windows is always the weird one.